### PR TITLE
Mode vis

### PR DESCRIPTION
**Pull request by blin4444**
Replay mode-vis commits in this repository
Quick fix for right axis bug

### CODE REVIEW

File: lib/timeline/component/LineGraph.js [hunk 1]
```diff
@@ -667,11 +666,13 @@ LineGraph.prototype._updateGraph = function () {
 
       //  at changeCalled, abort this update cycle as the graph needs another update with new Width input from the Redraw container.
       //  Cleanup SVG elements on abort.
-      if (changeCalled == true) {
+
+      /// [MODE change] remove this "optimization" so graph doesn't disappear when zooming in/out in browser
```
Comment (honteng) [inline lib/timeline/component/LineGraph.js:670]: how about the penalty?

File: lib/timeline/component/LineGraph.js [hunk 2]
```diff
@@ -471,18 +473,15 @@ LineGraph.prototype._updateAllGroupData = function (ids, groupIds) {
           }
         }
       }
-    }
+    } */
 
     //Now insert data into the arrays.
-    for (i = 0; i < items.length; i++) {
```
Comment (honteng) [inline lib/timeline/component/LineGraph.js:477]: oh, the original code is pretty bad :-(

Comment (blin4444) [reply lib/timeline/component/LineGraph.js:477]: it seemed to be reusing item/groupId from above.

File: lib/timeline/component/LineGraph.js (hunk 1 above)
Comment (blin4444) [reply lib/timeline/component/LineGraph.js:670]: unsure. But this was my fix from before; I am just moving it here.

**Pull request merged**